hms-app/src/components/admin-tools/UserForm.tsx

```diff
@@ -0,0 +1,197 @@
+import {
```

Comment:
this might be better named as `EditUserForm` since it's purpose is specifically to do that

Reply:
it might also make sense located outside the `admin-tools` folder as likely we would allow users to edit their own information which they won't need to be an Admin to do :thinking: